contracts/cw1-whitelist-ng/src/query.rs

```diff
@@ -1,57 +1,47 @@
 // This whole thing is definitely to be easly generated from trait itself
 
-use cosmwasm_std::{Addr, CosmosMsg, CustomQuery, QuerierWrapper, StdResult};
+use cosmwasm_std::{Addr, CosmosMsg, QuerierWrapper, StdResult};
 use cw1::CanExecuteResponse;
 use serde::Serialize;
 
-use crate::msg::{AdminListResponse, Cw1QueryMsg, WhitelistQueryMsg};
+use crate::msg::{self, AdminListResponse};
 
 #[must_use]
-pub struct Cw1Querier<'a, C>
-where
-    C: CustomQuery,
-{
+pub struct Cw1Querier<'a> {
```

Comment:
Why choice to remove `C` here?
Not an objection, just curious if it was design choice, or too difficult to implement for now?
(As you did include generic `T` on messages)

Reply:
On the other hand, there is no need here for it, so no big deal...

Reply:
Because I inittially messed it up, and now I don't yet support custom query types - only custom message types (I initially supported them both but in the way they had to be the same type which was incorrect). As it is not critical for this very contract, I leave it as it is, but custom queries are definitely to be supported (it is not even hard - the same way as custom messages are supported, just realising it is distinct thing).